pandas/tests/reshape/test_pivot.py

```diff
@@ -2153,3 +2153,40 @@ def test_pivot_index_none(self):
 
         expected.columns.name = "columns"
         tm.assert_frame_equal(result, expected)
+
+    def test_pivot_index_list_values_none_modifies_args(self):
+        # GH37635
```

Comment:
This PR is about making sure that the index list does not change.
Would it be more reasonable to test here only that, without comparing pivoted table with the expected one?

Reply:
I had that originally, but a previous review asked that I 
> check here that you got the correct result from the pivot (might need to hard-code)

Reply:
I don't feel strongly on this
But I'd say it's useful to check that the pivot is doing what it's supposed to before checking that the args aren't changed

Reply:
I looked through the test module and noticed that there are no tests on ``pivot`` when index is a list and values are None.
Thus, there is no repetition of any kind and it is perfectly fine to have the test as it is.
Thank you!